### PR TITLE
[+] Technical - On start, create a forestadmin.json file that contains the apimap definition

### PR DESCRIPTION
**Pull request by VincentMolinie**
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request
- [x] Write changes made in the CHANGELOG.md
- [x] Test manually the implemented changes
- [ ] Review my own code (indentation, syntax, style, simplicity, readability)
- [ ] Wonder if you can improve the existing code

### CODE REVIEW

**Comment by VincentMolinie** [comment]
[(5) Liana - On start, Express liana should create a forestadmin.json file that contains the apimap definition.](https://trello.com/c/zDhyjFv2/3102-5-liana-on-start-express-liana-should-create-a-forestadminjson-file-that-contains-the-apimap-definition)

**Comment by VincentMolinie** [comment]
[(5) Liana - On start, Express liana should create a forestadmin.json file that contains the apimap definition.](https://trello.com/c/zDhyjFv2/3102-5-liana-on-start-express-liana-should-create-a-forestadminjson-file-that-contains-the-apimap-definition)

File: src/index.js [hunk 1]
```diff
@@ -210,6 +210,11 @@ exports.init = (Implementation) => {
           }
         });
 
+        if (process.env.NODE_ENV !== 'production') {
+          const filename = `${path.resolve('.')}/forestadmin.json`;
```
Comment (ArnaudValensi) [inline src/index.js]: The filename has chaged for `forestadmin-schema.json`

File: .eslint-bin/js-list.json [hunk 2]
```diff
@@ -2,7 +2,9 @@
   ".eslint-bin/lint-regitered-files.js",
   ".eslint-bin/pre-commit-hook.js",
   "src/deserializers/resource.js",
+  "src/generators/schemas.js",
   "src/index.js",
+  "src/integrations/stripe/serializers/invoices.js",
```
Comment (ArnaudValensi) [inline .eslint-bin/js-list.json]: This is not supposed to be here, but if the file is valid, let's keep it.

File: src/index.js [hunk 3]
```diff
@@ -195,7 +195,10 @@ exports.init = (Implementation) => {
           'check on Forest that you copied it properly in the Forest ' +
           'initialization?');
       } else if (opts.envSecret) {
-        const collections = _.values(Schemas.schemas);
+        const forestadminSchemaFilename = `${path.resolve('.')}/forestadmin-schema.json`;
+        const collections = process.env.NOD_ENV === 'production'
+          ? JSON.parse(fs.readFileSync(forestadminSchemaFilename))
```
Comment (ArnaudValensi) [inline src/index.js]: We should print a clear message if:
- the file is invalid
- the file is empty
- the file is not present

Comment (ArnaudValensi) [reply src/index.js]: After this it's all good.

**Pull request merged**